manager/controllers/app/blueprint_controller.go

```diff
@@ -250,6 +258,179 @@ func (r *BlueprintReconciler) applyChartResource(ctx context.Context, cfg *actio
 	return rel, nil
 }
 
+func execPod(k8sClient kubernetes.Interface, config *restclient.Config, podName string, namespace string,
```

Comment:
the method is look as a generic command executed on a Pod.
I'd check probably something similar exists, so we can reuse it.
if not, it can be moved into utils, it is not blueprint_controller specific.

Reply:
Moved to `utils`.